vm/src/stdlib/os.rs

```diff
@@ -184,6 +185,19 @@ impl FsPath {
             FsPath::Bytes(b) => b.as_bytes(),
         }
     }
+
+    pub fn as_str(&self) -> &str {
+        match self {
+            FsPath::Bytes(b) => std::str::from_utf8(b).unwrap(),
+            FsPath::Str(s) => s.as_str(),
+        }
+    }    
```

Comment:
Since `FsPath` is intended to obtain `OsString` in Rust side, I am opposed to add `as_str` like this signature here.

My suggestion is adding `fn to_os_string(&self) -> std::os::OsString` and use it from `fn compile`.

Not this case, but even if we need to implement `as_str()`, it will look like:
```rust
pub fn as_str(&self) -> Result<&str, Utf8Error>
```